plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/data/CameraServerData.java

```diff
@@ -0,0 +1,50 @@
+package edu.wpi.first.shuffleboard.plugin.cameraserver.data;
+
+import com.google.common.collect.ImmutableMap;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import edu.wpi.first.shuffleboard.api.data.ComplexData;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import javafx.scene.image.Image;
+
+// serialversion UID doesn't matter since this is only used in one JVM at a time
+@SuppressFBWarnings("SE_NO_SERIALVERSIONID")
+public final class CameraServerData extends ComplexData<CameraServerData> implements Serializable {
```

Comment:
This might be general enough that it should be in BasePlugin, so other non-CameraServer sources and widgets could use it

Reply:
This is why I wanted to bundle the builtin plugins separately, and why I gave them maven-style IDs, so that third parties can depend on our plugins.